src/environments/environment.ts

```diff
@@ -30,6 +30,10 @@ export const environment = {
   supportedLanguages: window['env']['supportedLanguages'] || 'en-US,es-MX,fr-FR',
   preloadClients: window['env']['preloadClients'] || true,
 
+  superset: {
+    enabled: window['env']['supersetEnabled'] || true
```

Comment:
Please have this functionality `disabled` by default. This seems to me an optional feature.

Reply:
It is disabled in the HTML file by using:! sign, so it will not shown by default

Reply:
It is still confusing... maybe you should name it as disabled then or something... but you understand where i am coming from right?

Here is a configuration which value is `true`... superset.enabled = true and when it is true, then the superset is disabled... thats...wrong :D